src/applications/vaos/services/healthcare-service/index.js

```diff
@@ -13,38 +13,53 @@ function parseId(id) {
 /**
  * Method to get available HealthcareService objects.
  *
- * @param {String} facilityId
- * @param {String} typeOfCareId
+ * @param {String} facilityId The VistA facility id
+ * @param {String} typeOfCareId An id for the type of care to check for the chosen organization
  * @param {String} systemId
+ * @param {Boolean} params.useVSP A flag that determines whether or not to use the new VSP apis
  *
  * @returns {Array} An a collection of HealthcareService objects.
  */
 export async function getAvailableHealthcareServices({
   facilityId,
   typeOfCareId,
   systemId,
+  useVSP,
 }) {
-  try {
-    const clinics = await getAvailableClinics(
-      parseId(facilityId),
-      typeOfCareId,
-      systemId,
-    );
+  if (useVSP) {
+    const results = await fhirSearch({
+      query:
+        `HealthcareService?location:Location.identifier=${facilityId}` +
+        '&characteristic=PATIENTDS_ENABLED',
+      mock: () =>
+        facilityId === '983'
+          ? import('./mock_healthcare_system_983.json')
+          : import('./mock_healthcare_system_984.json'),
+    });
+    return results.filter(item => item.resourceType === 'HealthcareService');
```

Comment:
You can leave out this `filter`, since all the items should be HealthcareService resources.

Reply:
Done.

Reply:
Not done. Removed the filter and the test failed. FHIR call should return 7 results but actually returned 10. Included 3 with resourceType='Location' Is the query correct?

HealthcareService?**_location:Location.identifier=${facilityId}_**&characteristic=PATIENTDS_ENABLED

Reply:
That query is fine, I think it's just that you need to use one of your new mock files in your test. This line is https://github.com/department-of-veterans-affairs/vets-website/pull/13582/files#diff-6bf037c41e89e6a89f161dc618786716R135 is using the other mocks, not your new ones.